docs/spec_changelog.md

```diff
@@ -0,0 +1,43 @@
+---
+id: "spec_changelog"
+title: "Twirp Wire Protocol Changelog"
+sidebar_label: "Changelog"
+---
+
+This document lists changes to the Twirp specification.
+
+## Changed in v6
+
+### URL scheme
+
+In [v5](./PROTOCOL.md), URLs followed this format:
+
+```bnf
+**URL ::= Base-URL "/twirp/" [ Package "." ] Service "/" Method**
+```
+
+Version 6 changes this format to remove the mandatory `"/twirp"` prefix, and
+changes `Package` and `Service` to be delimited with a `/` instead of a `.`
+character:
+
+```bnf
+**URL ::= Base-URL "/" [ Package "/" ] Service "/" Method**
+```
+
+Also, `Base-URL` can now contain a path component - in other words, it's legal
+to set any prefix you like.
+
+If you loved the old `/twirp` prefix, you can still use it by using a base URL
+that ends with `/twirp`. You're no longer forced into use it, however.
```

Comment:
Users can keep that prefix, but 1) they'll need to remove it before passing the request to the Twirp generated code, and 2) the rest of the URI path won't be the same as v5 because of the `.`->`/` change (unless the package is unset).

Reply:
Right, I'll clarify this.